src/main.zig

```diff
@@ -1,4 +1,5 @@
 const std = @import("std");
+const main_log = std.log.scoped(.main);
```

Comment:
I don't really like that I had to call this one main_log instead of just log like in the other files, since otherwise it would clash with the custom log implementation. I argued for renaming the log implementation to "logger" to prevent this name collision here https://github.com/ziglang/zig/pull/6039#issue-466729548:
> The intended usage pattern of `const log = std.log.scoped(.main); log.err("oops", .{});` can be used everywhere _except_ in the root source file, if that file overrides the log implementation (since it would result in a naming conflict). Is this a problem? Maybe the log implementation could be renamed to "logger" instead?

Do you have any ideas to improve this, or do you think it's fine the way it is?